cocoadocs.rb

```diff
@@ -408,7 +408,7 @@ def document_spec(spec)
       swift = cloc_results.find { |r| r[:language] == 'Swift' }
       header = cloc_results.find { |r| r[:language] == 'C/C++ Header' }
 
-      if swift && (!header || swift[:files] >= header[:files])
```

Comment:
at this point if you have _any_ swift, we may as well just call you a swift pod. Fixes some odd side effects.